warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -249,7 +249,7 @@ <h4 class="modal-title" id="exampleModalLabel">Remove role for {{ role.user.user
             {% for pub in oidc_publishers %}
             <tr>
               <td>{{ pub.publisher_name }}</td>
-              <td><a href="{{ pub.publisher_url }}">{{ pub.publisher_url }}</a></td>
+              <td><a href="{{ pub.repository_url }}">{{ pub.repository }}</a></td>
```

Comment:
Rather than rendering the slug, let's just render the whole URL here.

```suggestion
              <td><a href="{{ pub.repository_url }}">{{ pub.repository_url }}</a></td>
```